run/libreoffice2john.py

```diff
@@ -75,6 +75,9 @@ def process_file(filename):
                 data = element.get("{urn:oasis:names:tc:opendocument:xmlns:manifest:1.0}key-size")
                 if data:
                     key_size = data
+                data = element.get("{urn:oasis:names:tc:opendocument:xmlns:manifest:1.0}start-key-generation-name")
+                if data:
+                    start_key_generation_name = data
 
     if not is_encrypted:
         sys.stderr.write("%s is not an encrypted OpenOffice file!\n" % filename)
```

Comment:
If you are in the mood and agree, why not update all but `--help` from OpenOffice to LibreOffice.

Reply:
I'm not currently in the mood and would rather see someone do that separately. I was also thinking of opening an issue to replace all `sys.exit(-1)` in Python scripts with `sys.exit(1)` (some already do the latter).

Reply:
Not arguing with that, but what is the difference?
```
$ sh -c "exit 0" && echo good || echo bad
good
$ sh -c "exit 1" && echo good || echo bad
bad
$ sh -c "exit -1" && echo good || echo bad
bad
```

Reply:
> If you are in the mood and agree, why not update all but `--help` from OpenOffice to LibreOffice.

Is that better at all? Shouldn't it be something like `odf2john`?

Reply:
...and for end users it would obviously make most sense to just have a single `office2john` that caught all of them buggers

Reply:
`sys.exit(-1)` feels like abuse, forcing the larger integer -1 to be turned into 255. If that's the exit code we want, we'd need to cleanly spell it out as 255. But we already use simply 1 in programs where we don't have the -1 to 255 abuse.

The script was apparently called odf2john before and then renamed.

Reply:
I couldn't care less about politics and especially not in the Open Source world but in my mind "opendocument" is/was (or was intended to) be a standard. Libreoffice is not a standard: Good or bad, it's just a fork or branch of code. It might be the original one or the intended one but that's not even relevant for us. Please correct me if I'm wrong, I'm very ignorant in these matters.

Reply:
I think it's a matter of what's more technically correct (ODF) vs. what users look for (LibreOffice). I'm not aware of a good solution. @ch3root might want to weigh in on this.

Reply:
Users of OpenOffice would not likely look for LibreOffice. I can't think of a great solution either.

Reply:
I see now the format is correctly called `ODF` which makes it even stranger to call the tool `libreoffice2john.py`. We could make a symlink... but let's not.